docs: mention .env (#1463)
vuejs/vuefire: docs/nuxt/getting-started.md

### Configuring the Admin SDK

> If you are using SSR with any auth related feature, you will need to create a [service account](https://firebase.google.com/support/guides/service-accounts) and provide its content as an _environment variable_ named `GOOGLE_APPLICATION_CREDENTIALS`. This should be kept in a .env file. 

Reviewer: ```suggestion
If you are using SSR with any auth related feature, you will need to create a [service account](https://firebase.google.com/support/guides/service-accounts) and provide its content as an _environment variable_ named `GOOGLE_APPLICATION_CREDENTIALS` in the `.env` file. 
```
